engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/simple/SimpleProcessStateDefinitionManager.scala

```diff
@@ -51,7 +53,7 @@ object SimpleProcessStateDefinitionManager extends ProcessStateDefinitionManager
     SimpleStateStatus.DuringCancel -> "The process currently is being canceled.",
     SimpleStateStatus.Failed -> "There are some problems with checking state of process..",
     SimpleStateStatus.Finished -> "The process completed successfully.",
-    SimpleStateStatus.Error -> "There are some errors with process state. Please check if everything is okay with process!"
+    SimpleStateStatus.Error -> "There are some errors with process state. Please check if everything is okay with process!",
```

Comment:
You don't missed something here? Looks like you didn't handle `RunningError`

Reply:
I've added a missing tooltip and description.